client/devdocs/design/index.jsx

```diff
@@ -9,6 +9,7 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { slugToCamelCase } from 'devdocs/docs-example/util';
 import { trim } from 'lodash';
+import Gridicons from 'gridicons/cjs/example';
```

Comment:
Since we're doing it already for `social-logos`, let's add an alias for Gridicon to our webpack config (as seen [here](https://github.com/Automattic/wp-calypso/pull/23086/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR171)).

Reply:
(That will allow us to `import Gridicons from 'gridicons/example';` here.)

Reply:
Done at 166ed8a